invenio_app/helpers.py

```diff
@@ -46,6 +49,60 @@ def get_safe_redirect_target(arg='next', _target=None):
     return None
 
 
+class ThemeJinjaLoader(BaseLoader):
```

Comment:
Should we call it ``PrefixJinjaLoader`` instead (same as the description below?)

Reply:
There's already a [`jinja.PrefixLoader`](https://jinja.palletsprojects.com/en/2.11.x/api/#jinja2.PrefixLoader) that does the opposite of what we need though (i.e. you call `render_template('bootstrap3/invenio_theme/page.html')` and it splits the prefix and search into custom places, not so useful in our case)... 

I went back and forth in the beginning with making a generic loader and sub-classing to allow getting the `APP_THEME` variable from `app.config`, but it seemed like overkill... In the end, it feels really customized so I went with explicitly naming like this